packages/client/src/editorApp/config.ts

```diff
@@ -52,8 +52,8 @@ export interface EditorAppConfig {
     domReadOnly?: boolean;
     readOnly?: boolean;
     overrideAutomaticLayout?: boolean;
-    editorOptions?: monaco.editor.IStandaloneEditorConstructionOptions;
-    diffEditorOptions?: monaco.editor.IStandaloneDiffEditorConstructionOptions;
+    editorOptions?: monaco.editor.IStandaloneEditorConstructionOptions | monaco.editor.IEditorOptions;
```

Comment:
What is the point of this change? IStandaloneEditorConstructionOptions already extends IEditorConstructionOptions which extends IEditorOptions :thinking: (and all fields are optional)